org.idempiere.test/pom.xml

```diff
@@ -133,6 +133,54 @@
 					</dependency-resolution>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>com.googlecode.maven-download-plugin</groupId>
+				<artifactId>download-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>download-test-resource-poi-ooxml.xlsx</id>
+						<phase>validate</phase>
+						<goals>
+							<goal>wget</goal>
+						</goals>
+						<configuration>
+							<!--${url.file.srv}/database/11/Adempiere_pg.jar-->
+							<url>https://raw.githubusercontent.com/hieplq/binary.file/IDEMPIERE-6117/test/poi-ooxml.xlsx</url>
+							<unpack>false</unpack>
+							<outputDirectory>resources</outputDirectory>
+							<retries>5</retries>
+						</configuration>
+					</execution>
+					<execution>
+						<id>download-test-resource-poi-5.2.3</id>
+						<phase>validate</phase>
+						<goals>
+							<goal>wget</goal>
+						</goals>
+						<configuration>
+							<!--${url.file.srv}/database/11/Adempiere_pg.jar-->
+							<url>https://raw.githubusercontent.com/hieplq/binary.file/IDEMPIERE-6117/test/poi-5.2.3.jar</url>
+							<unpack>false</unpack>
+							<outputDirectory>resources</outputDirectory>
+							<retries>5</retries>
+						</configuration>
+					</execution>
+					<execution>
+						<id>download-test-resource-poi-ooxml-5.2.3</id>
+						<phase>validate</phase>
+						<goals>
+							<goal>wget</goal>
+						</goals>
+						<configuration>
+							<!--${url.file.srv}/database/11/Adempiere_pg.jar-->
+							<url>https://raw.githubusercontent.com/hieplq/binary.file/IDEMPIERE-6117/test/poi-ooxml-5.2.3.jar</url>
+							<unpack>false</unpack>
+							<outputDirectory>resources</outputDirectory>
+							<retries>5</retries>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
```

Comment:
this is unnecessary, just add the test xlsx to test project. it is just 1 file and I don't expect we need to make frequent update of it either.

Reply:
In addition to the XLSX file, I also have two JAR files for testing case updates and multi-version installation.

Reply:
but that's not what iDempiere core is using, we shouldn't add test here that's for your project work.

Reply:
Also, for multi version, AFAIU, if you don't limit the import version in MANIFEST.MF, the resolver should always resolve to the latest version.

Reply:
The test above doesn't test anything for multi version of POI. You install a different version and unless your bundle specifically import only that version, the OSGi resolver will just resolve to the latest version install.